core/config/envs.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import "math/big"
+
+type ChainEnv string
+
+const (
+	HoleskyEnv  = ChainEnv("holesky")
+	EthereumEnv = ChainEnv("ethereum")
+)
+
+var (
+	MainnetChainID  = big.NewInt(1)
+	CurrentChainEnv = ChainEnv("ethereum")
+)
+
+func IsMainnet() bool {
+	return CurrentChainEnv == HoleskyEnv
+}
```

Comment:
I would build a map here for scalability, returning the URL based on the `ChainEnv` value. 

In a few months we would have more chains to support, so the function name `IsMainnet` will become ambiguous.